components/auth/FormField.tsx

```diff
@@ -0,0 +1,55 @@
+import Ionicons from "@expo/vector-icons/Ionicons";
+import React, { useState } from "react";
+import { View, Text, TextInput, TextInputProps, TouchableOpacity } from "react-native";
+
+interface FormInputGroups {
+  title: string;
+  value: string;
+  placeholder: string;
+  handleChangeText: (e: string) => void;
+}
+
+type CustomInputGroups = TextInputProps & FormInputGroups;
+
+const FormField = ({
+  title,
+  value,
+  placeholder,
+  handleChangeText,
+  ...props
+}: CustomInputGroups) => {
+  const [showPassword, setShowPassword] = useState(false)
+
+  return (
+    <>
+      <View className="space-y-2 mt-7">
+        <Text className="text-base text-neutral-100">{title}</Text>
+        <View className="w-full border-neutral-600 border-2 focus:border-neutral-400 justify-center h-16 my-4 rounded-2xl bg-neutral-700 flex-row">
+          <TextInput
+            value={value}
+            placeholder={placeholder}
+            placeholderTextColor={"#fafafa"}
```

Comment:
Maybe you want to create a component for this. I can imagine you having to put a property colour in every text input you have.

You can have a custom component where you set that a colour by default and you can optionally pass those colours if you want.

Reply:
I am not clear on this. Should I make the color dynamic or the entire TextInput? Like passing the color as a prop?

Reply:
Done

Reply:
He means you should create a component for text inputs and then call it every time you need it